.github/workflows/build-nuget.yml

```diff
@@ -0,0 +1,106 @@
+name: Package NuGet
+
+on:
+    push:
+        tags:
+            - '*'
+    workflow_dispatch:
+        inputs:
+            nugetPackageVersion:
+                description: "NuGet package version"
+                required: true
+            shouldCreateTag:
+                description: "Create a git tag + Release?"
+                required: true
+                type: boolean
+                default: true
+
+permissions:
+    contents: write # "write" needed for the "Tag a release" step.
+
+jobs:
+    compile-openal:
+        name: Build OpenAL-CS
+        uses: ./.github/workflows/build-openal.yml
+
+    compile-natives:
+        name: Build native dependencies
+        uses: ./.github/workflows/build-natives.yml
+
+    package-nuget:
+        name: Package NuGet
+        runs-on: ubuntu-22.04
+        needs: [compile-openal, compile-natives]
+        steps:
+            - name: Clone repository
+              uses: actions/checkout@v3
+
+            - name: Prepare environment variables
+              run: |
+                if [ "${{ github.event_name }}" = "push" ]; then
+                    echo "The git tag is ${GITHUB_REF#refs/tags/}"
+                    echo "PACKAGE_VERSION=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+                    echo "SHOULD_TAG_RELEASE=true" >> $GITHUB_ENV
+                else
+                    echo "This was ran manually with nuget package version ${{ github.event.inputs.nugetPackageVersion }}"
```

Comment:
If this is (important to be) logged here, should the nuget package version also appear when pushing a tag?

Reply:
Oh derp, that is leftover debug logging.
The git tag, release name and nuget package version are all the same thing btw.